libs/blocks/instagram/instagram.js

```diff
@@ -1,9 +1,10 @@
 import { createIntersectionObserver, createTag, loadScript } from '../../utils/utils.js';
 
 export default function init(a) {
+  if (a.parentElement.firstChild.nodeType === Node.TEXT_NODE) return;
```

Comment:
nit: move to a util in utils.js. Something like 
`const isParentText = (node) => node.parentElement.firstChild.nodeType === Node.TEXT_NODE`